examples/rgbpp_transfer.py

```diff
@@ -0,0 +1,35 @@
+from rgbpp.types import RgbppTransferReq
+from rgbpp.rpc import rpc
+
+# Warning: The example is not ready which inlcudes todo list to be resolved
+def transfer_rgbpp_on_btc(params: RgbppTransferReq):
+  response = rpc.generate_rgbpp_transfer_tx(params)
+  btc_psbt_hex = response['btc_psbt_hex']
+  ckb_virtual_tx_result = response['ckb_virtual_tx_result']
+
+  # TODO: pseudocode
+  # psbt = PSBT.from_hex(btcPsbtHex)
+  # psbt.sign_all_inputs(btc_key_pair);
+  # psbt.finalize_all_inputs();
+  # const btcTx = psbt.extractTransaction();
+  # btc_tx_id = btc_service.send_btc_transaction(psbt.extract_transaction().to_hex());
```

Comment:
According to the tests of `buidl`, I think this part of the code should look like this:
```python
hex_psbt = "..."
psbt_obj = PSBT.parse(BytesIO(bytes.fromhex(hex_psbt)))
psbt_obj.tx_obj.network = "testnet"
psbt_obj.sign(signer)
psbt_obj.finalize()
tx_obj = psbt_obj.final_tx()
tx_hex = tx_obj.serialize().hex()
```

ref: https://github.com/buidl-bitcoin/buidl-python/blob/main/buidl/test/test_psbt.py#L190-L213

Reply:
How to generate transaction hex while excluding `witness`:
```py
tx_hex = tx_obj.serialize_legacy().hex()
```